galley/pkg/server/settings/args.go

```diff
@@ -127,7 +127,8 @@ type Args struct { // nolint:maligned
 	EnableProfiling bool
 	PprofPort       uint
 
-	UseOldProcessor bool
+	UseOldProcessor             bool
+	EnableFsnotifySourceWatcher bool
```

Comment:
Please move the flag closer to the other config related permanent flags (e.g. under DisableResourceReadyCheck) and comments.